verification.py

```diff
@@ -664,6 +664,70 @@ def run_dtwf_coalescent_comparison(self, test_name, **kwargs):
             pyplot.savefig(f, dpi=72)
             pyplot.close('all')
 
+
+    def run_equivalent_models_comparison(self, test_name, kwargs1, kwargs2):
+        df = pd.DataFrame()
+        for i, kwargs in enumerate([kwargs1, kwargs2]):
+            print("Running: ", kwargs)
+            replicates = msprime.simulate(**kwargs)
+            data = collections.defaultdict(list)
+            for ts in replicates:
+                t_mrca = np.zeros(ts.num_trees)
+                for tree in ts.trees():
+                    t_mrca[tree.index] = tree.time(tree.root)
+                data["tmrca_mean"].append(np.mean(t_mrca))
+                data["num_trees"].append(ts.num_trees)
+                data["param_set"].append(i)
+            df = df.append(pd.DataFrame(data))
+
+        basedir = os.path.join("tmp__NOBACKUP__", test_name)
+        if not os.path.exists(basedir):
+            os.mkdir(basedir)
+
+        df_1 = df[df.param_set == 0]
+        df_2 = df[df.param_set == 1]
+        for stat in ["tmrca_mean", "num_trees"]:
+            v1 = df_1[stat]
+            v2 = df_2[stat]
+            sm.graphics.qqplot(v1)
+            sm.qqplot_2samples(v1, v2, line="45")
+            f = os.path.join(basedir, "{}.png".format(stat))
+            pyplot.xlabel("Parameter set 2")
+            pyplot.ylabel("Parameter set 1")
+            pyplot.savefig(f, dpi=72)
+            pyplot.close('all')
+
+    def add_coalescent_diff_Ne(self):
+        """
+        Checks that setting Ne in the hudson coalescent has the expected
+        effect.
+        """
+        population_configurations = [
+            msprime.PopulationConfiguration(sample_size=10, initial_size=1000)]
+        num_replicates=1000
+        recombination_rate=0.1
+
+        kwargs1 = dict(
+                population_configurations=population_configurations,
```

Comment:
Sure enough, it does!

tmrca_mean:
![tmrca_mean](https://user-images.githubusercontent.com/8810687/42764433-9adb6140-88e3-11e8-97c3-4b1c5850f7c1.png)

This gets to the root of some of my confusion then - shouldn't we be able to specify both? At least in the DTWF, since we can't rescale time in the same way.

https://github.com/tskit-dev/msprime/blob/6d7893a6124c9e57617ec49f43111c627c477795/lib/msprime.c#L2264-L2270

Reply:
> This gets to the root of some of my confusion then - shouldn't we be able to specify both?

Yes, probably. I'm not entirely sure I know what it would mean in the Hudson case, but I'll confess to being a bit confused about how it all works!

Reply:
Same here! Ok I'll put a bit of thought into it...